Assets/Scenes/Game Flow/ATH_Lobby.unity

```diff
@@ -3402,7 +3402,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
```

Comment:
Why do you have this scene modified?

Reply:
That's weird, i didn't touch the scene... 

Lemme try to find the original file and make the changes back naooo